infra/build/functions/base_images.py

```diff
@@ -55,7 +57,40 @@ def _get_base_image_steps(images, tag_prefix=TAG_PREFIX):
             '-t',
             image,
             '-t',
-            f'{image}:{MAJOR_VERSION}',
+            f'{image}:{MAJOR_TAG}',
+            '.',
+        ],
+        'dir': 'oss-fuzz/infra/base-images/' + base_image,
+        'name': 'gcr.io/cloud-builders/docker',
+    })
+
+  return steps
+
+
+def _get_introspector_base_images_steps(images, tag_prefix=TAG_PREFIX):
+  """Returns build steps for given images version of introspector"""
+
+  steps = [{
+      'name':
+          'gcr.io/oss-fuzz-base/base-runner',
+      'args': [
+          'bash', '-c',
+          (f'sed -i s/base-clang/base-clang:{INTROSPECTOR_TAG}/g'
```

Comment:
let's set the replace pattern to 'base-clang:.*' in case the base-clang is already set to a tag.

Reply:
First, probably you meant 'base-clang.*', right?
Second, the replacement is applied on vanilla oss-fuzz clone, so there won't be an explicit tag for the image used by base-builder. Unless you are assuming in future we may have such tagging.

Reply:
applied.